### PR TITLE
modify customized tuner doc and fix typo in training service doc

### PR DESCRIPTION
**Pull request by J-shang**

### CODE REVIEW

File: docs/en_US/TrainingService/LocalMode.rst [hunk 1]
```diff
@@ -17,8 +17,9 @@ You have an implementation for MNIST classifer using convolutional layers, the P
 
 To enable NNI API, make the following changes:
 
-* Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
-* Get predefined parameters
+   1.1 Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
```
Comment (QuanluZhang) [inline docs/en_US/TrainingService/LocalMode.rst:20]: I found the first sentence of this file has typo: "In this tutorial, we will use the example in [~/examples/trials/mnist-tfv1] to explain how to create and run an experiment on local with NNI API."

File: docs/en_US/TrainingService/LocalMode.rst [hunk 2]
```diff
@@ -91,7 +93,7 @@ To run an experiment in NNI, you only needed:
 
 ..
 
-   A set of examples can be found in ~/nni/examples after your installation, run ``ls ~/nni/examples/trials`` to see all the trial examples.
+   You can download nni source code and a set of examples can be found in ~/nni/examples, run ``ls ~/nni/examples/trials`` to see all the trial examples.
```
Comment (QuanluZhang) [inline docs/en_US/TrainingService/LocalMode.rst:96]: `~/nni/examples` is not right. `nni/examples` might be better

File: docs/en_US/TrainingService/Overview.rst [hunk 3]
```diff
@@ -65,6 +65,6 @@ Step 1. **Validate config and prepare the training platform.** Training service
...
 
-.. Warning:: The working directory of trial command has exactly the same content as ``codeDir``, but can have a differen path (even on differen machines) Local mode is the only training service that shares one ``codeDir`` across all trials. Other training services copies a ``codeDir`` from the shared copy prepared in step 1 and each trial has an independent working directory. We strongly advise users not to rely on the shared behavior in local mode, as it will make your experiments difficult to scale to other training services.
+.. Warning:: The working directory of trial command has exactly the same content as ``codeDir``, but can have a different paths (even on different machines) Local mode is the only training service that shares one ``codeDir`` across all trials. Other training services copies a ``codeDir`` from the shared copy prepared in step 1 and each trial has an independent working directory. We strongly advise users not to rely on the shared behavior in local mode, as it will make your experiments difficult to scale to other training services.
```
Comment (QuanluZhang) [inline docs/en_US/TrainingService/Overview.rst:68]: `a different paths` -> `different paths`

File: docs/en_US/Tuner/InstallCustomizedTuner.rst [hunk 4]
```diff
@@ -32,19 +28,26 @@ This command build a whl file which is a pip installation source.
...
 
-``nnictl package install dist/demo_tuner-0.1-py3-none-any.whl``
+``pip install dist/demo_tuner-0.1-py3-none-any.whl``
+
+Register the customized tuner as builtin tuner:
+-----------------------------------------------
+
+Run following command:
+
+``nnictl algo register --meta meta_file.yml``
```
Comment (QuanluZhang) [inline docs/en_US/Tuner/InstallCustomizedTuner.rst:38]: how to write `meta_file.yml`?

Comment (J-shang) [reply docs/en_US/Tuner/InstallCustomizedTuner.rst:38]: In fact we have a doc called `How to register customized algorithms as builtin tuners, assessors and advisors`, and it has more details, I don't know if this doc is necessary.

File: docs/en_US/TrainingService/Overview.rst (hunk 3 above)
Comment (J-shang) [reply docs/en_US/TrainingService/Overview.rst]: fix it

File: docs/en_US/TrainingService/LocalMode.rst (hunk 2 above)
Comment (J-shang) [reply docs/en_US/TrainingService/LocalMode.rst]: fix it

File: docs/en_US/TrainingService/LocalMode.rst (hunk 1 above)
Comment (J-shang) [reply docs/en_US/TrainingService/LocalMode.rst:20]: [nni/examples/trials/mnist-tfv1] and NNI APIs?

**Review by chicm-ms — APPROVED** [review]
Looks good for me.

File: docs/en_US/Tuner/InstallCustomizedTuner.rst (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/Tuner/InstallCustomizedTuner.rst:38]: what is the path of this doc?

File: docs/en_US/TrainingService/LocalMode.rst (hunk 1 above)
Comment (QuanluZhang) [reply docs/en_US/TrainingService/LocalMode.rst:20]: In this tutorial, we will use the example in [~/examples/trials/mnist-tfv1] to explain how to create and run an experiment on local with NNI API.

File: docs/en_US/Tuner/InstallCustomizedTuner.rst (hunk 4 above)
Comment (J-shang) [reply docs/en_US/Tuner/InstallCustomizedTuner.rst:38]: docs/en_US/Tutorial/InstallCustomizedAlgos.rst
https://nni.readthedocs.io/en/v2.0/Tutorial/InstallCustomizedAlgos.html

Comment (QuanluZhang) [reply docs/en_US/Tuner/InstallCustomizedTuner.rst:38]: yeah, it is duplicated, suggest to merge content of the two files together

File: docs/en_US/TrainingService/LocalMode.rst (hunk 1 above)
Comment (ultmaster) [inline docs/en_US/TrainingService/LocalMode.rst:20]: Might as well remove the indentation. As indentation is rst means "quote" I think.

Comment (J-shang) [reply docs/en_US/TrainingService/LocalMode.rst]: fix it

File: docs/zh_CN/hpo_advanced.rst [hunk 5]
```diff
@@ -9,4 +9,3 @@
     编写新的 Advisor <Tuner/CustomizeAdvisor>
     编写新的训练平台 <TrainingService/HowToImplementTrainingService>
     安装自定义的 Tuner，Assessor，Advisor <Tutorial/InstallCustomizedAlgos>
-    如何将自定义的 Tuner 安装为内置 Tuner <Tuner/InstallCustomizedTuner.md>
```
Comment (QuanluZhang) [inline docs/zh_CN/hpo_advanced.rst:12]: do not modify chinese doc

File: docs/en_US/Tuner/BuiltinTuner.rst [hunk 6]
```diff
@@ -52,7 +52,7 @@ Usage of Built-in Tuners
 
 Using a built-in tuner provided by the NNI SDK requires one to declare the  **builtinTunerName** and **classArgs** in the ``config.yml`` file. In this part, we will introduce each tuner along with information about usage and suggested scenarios, classArg requirements, and an example configuration.
 
-Note: Please follow the format when you write your ``config.yml`` file. Some built-in tuners need to be installed using ``nnictl package``\ , like SMAC.
+Note: Please follow the format when you write your ``config.yml`` file. Some built-in tuners have dependencies need to be installed using ``pip install nni[<tuner>]``, like SMAC's dependencies can be installed using ``pip install nni[SMAC]``.
```
Comment (QuanluZhang) [inline docs/en_US/Tuner/BuiltinTuner.rst:55]: "have dependencies need" -> "have dependencies that need"

File: docs/en_US/Tuner/BuiltinTuner.rst [hunk 7]
```diff
@@ -192,11 +192,11 @@ SMAC
 
 **Installation**
 
-SMAC needs to be installed by following command before the first usage. As a reminder, ``swig`` is required for SMAC: for Ubuntu ``swig`` can be installed with ``apt``.
+SMAC has dependencies need to be installed by following command before the first usage. As a reminder, ``swig`` is required for SMAC: for Ubuntu ``swig`` can be installed with ``apt``.
```
Comment (QuanluZhang) [inline docs/en_US/Tuner/BuiltinTuner.rst:195]: has dependencies that need to be

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**